editor/src/messages/tool/transform_layer/transform_layer_message_handler.rs

```diff
@@ -6,8 +6,8 @@ use crate::messages::tool::common_functionality::shape_editor::ShapeState;
 use crate::messages::tool::utility_types::{ToolData, ToolType};
 
 use document_legacy::layers::style::RenderData;
-
 use glam::DVec2;
+use graphene_core::vector::ManipulatorPointId;
```

Comment:
Move this up to the `use document_legacy::.` block above, and put a blank line between that block and the `use glam::...` line.

Reply:
The import order doesn't really change anything @Keavon - you either need to create a pr for rust format or remove it from the formatting rules.